junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSource.java

```diff
@@ -94,4 +94,6 @@
 	@API(status = EXPERIMENTAL, since = "5.5")
 	String emptyValue() default "";
 
+	String[] nullSymbol() default {};
```

Comment:
- `@API(status = EXPERIMENTAL, since = "5.5")` is missing.
- As the return type is an array of `String`, the method should be called `nullSymbols`, ending with a `s` ... 🤔

Reply:
Thanks for the reply!
I wasn't really sure about the `@API` Annotation. I guess this is the following check from the "Definition of Done"?

>  Method preconditions are checked and documented in the method's Javadoc

About the plural of `nullSymbol`: You are totally right!